### DISCUSSION TITLE
SIP-0009 Stackathon Stackchain Memes

### DISCUSSION THREAD

**Discussion by von-Hodl** [stackchainsats/stackchain#16] [General]
Rough draft) Stackathon Stackchain Memes

Stackchain is the mother meme from whose loins flows the richest memes in the history of memecraft.

In a public discussion found [here](https://twitter.com/i/spaces/1OyKAVPojrWGb) Stackchainers agreed that documenting the memes is critical.

What is a meme?

You might think you know the answer to this question. However if you have not drank the sweet nectar of stackchain memes crafted by the finest memelords sharping their craft in glory of stackhalla, do you even know what a meme is?

The history of stackchain has been etched in finest memes.

Problem to be solved.

So many memes.

The communities proposal is to create an automated solution to organize the dankest of memes so they are not lost.

**Comment by derekross** [comment]
SIP-0006 already exists. This would need to be SIP-0009.

**Reply by derekross** [reply]
Renamed to SIP-0009

**Reply by von-Hodl** [reply]
This idea is being moved to Sip-0008 as part of The StackChain Dashboard
